tests/gcp/hooks/test_bigquery_system.py

```diff
@@ -32,7 +32,7 @@ def setUp(self):
         self.instance = hook.BigQueryHook()
 
     @mock.patch(
```

Comment:
Why do we do mocking in system tests?

Reply:
Excellent point! I've verified are tests working without and removed them.

Reply:
Any progress?